src/Codeception/Util/Template.php

```diff
@@ -41,11 +47,14 @@ public function place($var, $val)
      *
      * @param array $vars
      */
-    public function setVars(array $vars)
+    public function setVars(array $vars): void
     {
         $this->vars = $vars;
     }
 
+    /**
+     * @return mixed|void
+     */
     public function getVar($name)
```

Comment:
$name is a string.

I don't like `void` in return type at all, I think that it should be assumed that it returns `null` when there is no matching element, but it is part of `mixed`.